docs/_js/ErrorCodesComponent.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/* global React ReactDOM errorMap:true */
+'use strict';
+
+function replaceArgs(msg, argList) {
+  let argIdx = 0;
+  return msg.replace(/%s/g, function() {
+    const arg = argList[argIdx++];
+    return arg === undefined ? '[missing argument]' : arg;
+  });
+}
+
+function segmentify(str) {
+  const urlRegex = /(https:\/\/fb\.me\/[a-z\-]+)/g;
```

Comment:
Yeah, sometimes we have a link in a message that points to a gist/doc page (e.g., [this one](https://github.com/facebook/react/blob/master/scripts/error-codes/codes.json#L121)). This function is used to "urlify" the link. See http://keyanzhang.github.io/react/docs/error-codes.html?invariant=119&args="foo"&args="bar" as an example.

I tried to use `dangerouslySetInnerHTML`, but it didn't work since we have stuff like `<%s>` in our error messages.